### PR TITLE
Add `GettextSetup::VERSION`

### PR DESCRIPTION
**Pull request by joshuaspence**
Adds a `GettextSetup::VERSION` constant. See puppetlabs/puppet#5822 for additional context.

### CODE REVIEW

File: lib/gettext-setup/version.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module GettextSetup
+  VERSION = '0.24'
```
Comment (smcclellan) [inline lib/gettext-setup/version.rb]: Is there a reason not to derive this from the git tags, like the gemspec did before?

Comment (joshuaspence) [inline lib/gettext-setup/version.rb]: I'm not sure that would work? As in, if I run `gem install gettext-setup`, then the git metadata won't be installed locally, right?

Comment (smcclellan) [inline lib/gettext-setup/version.rb]: I've been looking for a way to avoid maintenance commits to update the version prior to every release. I'm wondering if there's a way to have this constant be a little more dynamic in any way.

Backing up, what is the goal for adding this constant?

Comment (joshuaspence) [inline lib/gettext-setup/version.rb]: You were too quick, I just submitted another pull request which should justify what I am trying to achieve here, see puppetlabs/puppet#5822.

**Comment by joshuaspence** [comment]
See discussion in puppetlabs/puppet#5822.
